# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

MoveToPositionActionGoal action_goal
MoveToPositionActionResult action_result
MoveToPositionActionFeedback action_feedback
